RecoBTag/CTagging/python/RecoCTagging_cff.py

```diff
@@ -0,0 +1,17 @@
+from RecoBTag.CTagging.pfInclusiveSecondaryVertexFinderCvsLTagInfos_cfi import *
```

Comment:
For some reason I do not fully understand, I need the two definitions to be separate. Otherwise the default tests running in unscheduled mode will fail. The PAT sequence will try to re-build the TagInfos instead of taking what is provided by the AOD event content